lib/u3d/iniparser.rb

```diff
@@ -49,6 +46,10 @@ def load_ini(version, cached_versions, os: U3dCore::Helper.operating_system, off
         Utils.ensure_dir(default_ini_path)
         ini_path = File.expand_path(ini_name, default_ini_path)
         unless File.file?(ini_path)
+          if os == 'linux'
+            UI.error "No INI file for version #{version}"
```

Comment:
Well one can, but the main reason why I included that is to have some sort of failsafe if for some reason the user didn't have the INI file on his system. There are very few cases where this could happen, but I believe it's worth checking nonetheless.